druid-shell/src/common_util.rs

```diff
@@ -48,3 +50,33 @@ impl<F: FnOnce(&dyn Any) + Send> IdleCallback for F {
         (*self)(a)
     }
 }
+
+/// An incrementing counter for generating unique ids.
```

Comment:
Probably should write down the invariant that it can't be 0, though it's pretty clear from the safety requirement.